db/seeds.rb

```diff
@@ -6,12 +6,11 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 Product.create([
-                {title: 'Pasta Carbonara', description: 'Second courses', price: 9.00}, 
-                {title: 'Chicken Soup', description: 'Entrees', price: 13.00}, 
-                {title: 'Vitamin Salad', description: 'Salads', price: 7.00}, 
-                {title: 'Pasta With Walnut', description: 'Second courses', price: 25.00}, 
-                {title: 'Item Four', description: 'Second courses', price: 87.00}, 
-                {title: 'Item Five', description: 'Second courses', price: 57.00}, 
-                {title: 'Item Six', description: 'Second courses', price: 102.00}
+                { title: 'Pasta Carbonara', description: 'Second courses', price: 9.00 }, 
```

Comment:
very good that you used seeds 👏 

the spacing looks a little bit off though, you only need 2 spaces to indent the contents of the array:
```ruby
Product.create([
  { Product 1 params },
  { Product 2 params },
  etc..
])
```